pwa/src/components/alert/alert.tsx

```diff
@@ -15,8 +15,8 @@ const AlertComponent = () => {
     return (
       <div className={`conduction-alert conduction-alert-${alert.type}`}>
         {
-          alert.title && (
-            <h5>{alert.title}</h5>
+          alert.type === 'danger' && (
+            <h5><span>Oops something went wrong</span></h5>
```

Comment:
Because this is not a `sub-title` of a (e.g.) article or some other hierarchal form of content, it doesn't have to be a `h5` tag (or any other `h` tag). These are reserved to bring hierarchy into content and this pop-up is not part of that (or any) hierarchy.

Resulting in that this can just be a `<span>` without anything else.

Note: this could mean that we have to write some `css` to style it.

Reply:
Also, right now we're writing this syntax pretty complex, which I think is unnecessary. This would do the same (and would comply with out prettier):

```
{alert.type === "danger" && <span>Oops something went wrong</span>}
```

Reply:
Also, right now we're writing this syntax pretty complex, which I think is unnecessary. This would do the same (and would comply with out prettier):

```
{alert.type === "danger" && <span>Oops something went wrong</span>}
```